stt-worker/acquire.py

```
@@ -0,0 +1,106 @@
+    return {
+        "outtmpl": os.path.join(out_dir, "%(id)s.%(ext)s"),
+        "quiet": True,
+        "no_warnings": True,
+        "noprogress": True,
+        **_cookie_opts(),
+    }
+
+
+def _download_media(url: str, out_dir: str) -> str:
+    """CDN 직다운. 확장자는 Content-Type(→URL경로)으로 결정해 영상/이미지 모두 처리."""
+    req = urllib.request.Request(url, headers={"User-Agent": "Mozilla/5.0"})
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)